doc/crm/Generic_SAI_Extensions_CRM.md

```diff
@@ -0,0 +1,70 @@
+# Generic SAI Extension Critical Resource Monitoring (CRM)
+
+## Table of Content
+- [Introduction](#Itroduction)
+- [Scope](#Scope)
+- [Requirements](#Requirements)
+- [Highlevel Design](#Highlevel-Design)
+- [Unit Test Automation](#Unit-Test-Automation)
+
+## Revision
+| Rev  |   Date    |                        Author                        | Change Description                 |
+| :--: | :-------: | :--------------------------------------------------: | :--------------------------------: |
+| 0.1  | 1/17/2023 | Intel                                                |  Initial Revision                  |
+
+## Introduction <a name="Introduction"></a>
+High Level Design for critical resource monitoring (crm) in SONiC for Generic SAI Extension tables.
+
+## Scope <a name="Scope"></a>
+The scope of the design is to publish CRM counts, for Generic SAI Extension tables, into the COUNTERS-DB. Support for existing generic crm configurations that is applicable to all resources, like polling interval, is in the scope of this document. No additional configuration specific to extension tables resource is in the scope.

```

Comment:
- It looks these SAI extensions specific to PINS project? is that true?

Reply:
that's correct